app/uk/gov/hmrc/pegaproofofconceptfrontend/controllers/InputController.scala

```diff
@@ -49,7 +52,10 @@ class InputController @Inject() (
       },
       (validFormData: StringInputForm) => {
         logger.info(s"SUBMITTED STRING: '${validFormData.string}' TO PEGA")
-        Future.successful(Redirect(uk.gov.hmrc.pegaproofofconceptfrontend.controllers.routes.InputController.getStringInput))
+        pegaProxyConnector.submitPayloadToProxy(Payload.fromStringInputForm(validFormData)).map {
+          case response if response.status === 200 => Ok(views.fakePegaPage())
```

Comment:
it'd make more sense to move the 
```scala
logger.info(s"SUBMITTED STRING: '${validFormData.string}' TO PEGA")
```
logging in this case of the pattern match since it's talking about a successful submission to pega in the past tense